source/base/Text.ooc

```diff
@@ -21,8 +21,20 @@ import math
 
 Text: cover {
 	_buffer: TextBuffer
-	count ::= this _buffer count
-	isEmpty ::= this count == 0
+	count: Int {
+		get {
+			result := this _buffer count
+			this free(Owner Receiver)
+			result
+		}
+	}
```

Comment:
```ooc
count ::= {
    result := this _buffer count
    this free(Owner Receiver)
    result
}
```
would be cool too. I don't think that's supported in rock, but it shouldn't be too hard to make it possible, in that case.